pom.xml

```diff
@@ -10,7 +10,24 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.baloise.confluence</groupId>
     <artifactId>digital-signature</artifactId>
-    <version>7.0.5</version>
+    <version>8.0.0</version>
+
+    <properties>
+        <confluence.version>9.2.3</confluence.version>
+        <confluence.data.version>9.2.3</confluence.data.version>
+        <amps.version>8.11.1</amps.version>
```

Comment:
Update amps to 9.3.3 (see https://developer.atlassian.com/server/framework/atlassian-sdk/amps-sdk-release-notes/)